src/System.Drawing.Primitives/tests/DataContractSerializerTests.cs

```diff
@@ -101,6 +117,24 @@ public static void DCS_RectangleF()
             }
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public static void DCS_RectangleF_NotNetFramework()
+        {
+            var objs = new RectangleF[]
+            {
+            new RectangleF(1.50001f, -2.5000f, 1.5000f, -2.5000f)
```

Comment:
Indent?

Reply:
Or maybe that's how VS does it

Reply:
Most of the badly formatted code here was just copied verbatim from the existing testcase that covers netfx.

A good bit of the code could do with cleanup/refactoring.